optuna/storages/_journal/file.py

```diff
@@ -0,0 +1,184 @@
+import abc
+from contextlib import contextmanager
+import errno
+import json
+import os
+import time
+from typing import Any
+from typing import Dict
+from typing import Iterator
+from typing import List
+from typing import Optional
+import uuid
+
+from optuna.storages._journal.base import BaseJournalLogStorage
+
+
+LOCK_FILE_SUFFIX = ".lock"
+RENAME_FILE_SUFFIX = ".rename"
+
+
+class JournalFileBaseLock(abc.ABC):
+    @abc.abstractmethod
+    def acquire(self) -> bool:
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def release(self) -> None:
+        raise NotImplementedError
```

Comment:
Could you add the docstring to each method in this class to fill in the description of the `acquire()` and `release()` methods in JournalFileSymlinkLock and JournalFileOpenLock?

<img width="1140" alt="Screenshot 2022-08-30 14 48 32" src="https://user-images.githubusercontent.com/5564044/187359360-aeb302e0-db45-4ee9-a0a8-5936889a6aad.png">

* https://output.circle-artifacts.com/output/job/4db460c6-d068-405d-a771-84ff61b3f0e9/artifacts/0/docs/build/html/reference/generated/optuna.storages.JournalFileSymlinkLock.html#optuna.storages.JournalFileSymlinkLock
* https://output.circle-artifacts.com/output/job/4db460c6-d068-405d-a771-84ff61b3f0e9/artifacts/0/docs/build/html/reference/generated/optuna.storages.JournalFileOpenLock.html#optuna.storages.JournalFileOpenLock

Reply:
https://github.com/optuna/optuna/pull/3854/commits/9c987a7374bb5f79d17a2f789656cd7afc549a2c
https://github.com/optuna/optuna/pull/3854/commits/1e89cda03f34fadff657feecd8a3cf88144390db